jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -49,19 +49,34 @@ You need:
 
 * Your GitLab self-managed instance URL, e.g. `https://example.circleci.com``.
 +
-Your self-managed instance must already contain at least one project. The authorization attempt will be unsuccessful if your instance does not have any projects.
+Your self-managed instance must already contain at least one project. The authorization attempt will be unsuccessful if your instance does not have any projects.  Note that the self-managed instance must be accessible via the public internet, **it cannot be behind a firewall**.
 
 * A link:https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html[personal access token]. This token must have the `api` scope.  
 
-* Your instance's SSH public host keys. You can retrieve this from your instance by running `ssh-keyscan example.circleci.com` and copying the command's output.
+* Your instance's SSH public host keys. You can retrieve this from your instance by running `ssh-keyscan <instance_url>`, for example, `ssh-keyscan test-gitlab.circleci.com`, and copying the command's output.
```

Comment:
can we make this its own linkable section so that we can have a direct link from the in-app intructions?

Reply:
It is! but there's something odd going on with the html conversion, I compared the JSON and the HTML and the id is ending up as `#user-content-known-hosts-input` somewhere along the way. We can use that anchor to link to for now, and I'll make a ticket to look into the addition of `user-content` as a bug in the platform I think

Reply:
@rsylvian would you agree this is a platform thing/bug?